### PR TITLE
Lomas values schema

### PR DESCRIPTION
**Pull request by LancelotMarti**

### CODE REVIEW

File: charts/lomas/values.yaml
Comment (LancelotMarti) [inline charts/lomas/values.yaml:1]: runtime_args.settings.develop_mode to False?

File: charts/lomas/templates/ingress.yaml [hunk 1]
```diff
@@ -33,12 +33,9 @@ spec:
   {{- end }}
   {{- if .Values.ingress.tls }}
   tls:
-  {{- with .Values.ingress.tls }}
     - hosts:
-      {{- range .hosts }}
-      - {{ . | quote}}
-      {{- end }}
-  {{- end}}
+        - {{ .Values.ingress.hostname | quote }}
```
Comment (LancelotMarti) [inline charts/lomas/templates/ingress.yaml:37]: not entirely sure. to discuss

**Review by PaulineMauryL — APPROVED** [review]

File: charts/lomas/values.yaml
Comment (PaulineMauryL) [reply charts/lomas/values.yaml:1]: excellente question

Comment (PaulineMauryL) [reply charts/lomas/values.yaml:1]: mmh, je pense que pour le moment et tester oui, mais quand on sort du insee_miscellaneous on mettra False

**Pull request merged**